bin/travis/setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# pwd is the git repo.
+set -e
+
+echo "Creating settings/local.py"
+cp fjord/settings/travis.py-dist fjord/settings/local.py
+
+echo "Creating test database"
+mysql -e 'CREATE DATABASE fjord CHARACTER SET utf8 COLLATE utf8_unicode_ci;'
+
+echo "Updating product details"
+python manage.py update_product_details
```

Comment:
Pretty sure they do. We have an events api that pulls from the product details data.

Having said that, I could be wrong. Might be worth doing a travis test without this line to know for sure.